marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/crudQueriesfromExplore.cy.tsx

```diff
@@ -203,16 +200,16 @@ describe("Save/manage queries scenarios, developer role", () => {
     browsePage.getSaveQueryButton().click();
     browsePage.getErrorMessage().should("contain", "You already have a saved query with a name of new-query");
     browsePage.getSaveQueryCancelButton().click();
-    // checking previous query name is set clicking save modal icon
+
     browsePage.getSaveModalIcon().scrollIntoView().click();
     browsePage.getEditSaveChangesFormName().invoke("val").should("contain", "new-query-2");
     browsePage.getEditSaveChangesCancelButton().click();
-    // checking previous query name is set clicking edit modal icon
```

Comment:
@vmendez-marklogic instead of removing all the comments completely for huge tests like these where it is explaining exactly what is being validated, can we change them to logs? It will be useful when trying to understand the scenario while debugging